cmd/namespace/delete.go

```diff
@@ -138,7 +137,9 @@ func (nc *Command) watchDelete(ctx context.Context, namespace string, k8sLogger
 	select {
 	case <-stop:
 		ctxCancel()
+		logsCtxCancel()
 		oktetoLog.Infof("CTRL+C received, exit")
+		oktetoLog.Information("CTRL+C received, cancelling context and logs context but operation will continue in background")
```

Comment:
```suggestion
		oktetoLog.Information("CTRL+C received, cancelling wait and logs streaming but operation will continue in background")
```

Reply:
Also, checking how it looks like



I would change the message error by something like: `Watching namespace deletion stopped: interrupt signa received`